Assets/Resources/Settings.gd

```diff
@@ -23,6 +23,7 @@ export var minimized_window_size : Vector2 = Vector2(1100, 650)
 
 # settings
 export var show_secs_dash : bool = true
+export var set_24h_time: bool = true
```

Comment:
this is more a question than a review asking for changes, is it better to name the variable set_24h.... or to simplify it into 24h_time?